tests/fixtures/annotations/xsdata.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Config xmlns="http://pypi.org/project/xsdata" version="22.1">
+  <Output maxLineLength="79">
+    <Package>tests.fixtures.annotations</Package>
+    <Format repr="true" eq="true" order="false" unsafeHash="false" frozen="false" slots="false" kwOnly="false">dataclasses</Format>
+    <Structure>filenames</Structure>
+    <DocstringStyle>reStructuredText</DocstringStyle>
+    <RelativeImports>false</RelativeImports>
+    <CompoundFields defaultName="choice" forceDefaultName="false">true</CompoundFields>
+    <ImportAnnotations>true</ImportAnnotations>
```

Comment:
Somehow I would like the wording `postponed` in the config name, like `postponedAnnotations`